src/index.html

```diff
@@ -1,13 +1,95 @@
+<!-- #region HEAD-->
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <title>Bose</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
```

Comment:
Consider using a crossorigin attribute for the 'preconnect' link tags to improve performance. Example: 
```<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>```